src/bun.js/api/bun.classes.ts

```diff
@@ -107,10 +107,12 @@ export default [
       signalCode: {
         getter: "getSignalCode",
       },
-
       exited: {
         getter: "getExited",
       },
+      pipe_fds: {
```

Comment:
lets not add this getter to the api

instead, lets add a `stdio` getter which returns an array of `[this.stdin, this.stdout, this.stderr, ...rest]`